numpy/distutils/ccompiler_opt.py

```diff
@@ -307,7 +312,7 @@ class attribute `conf_features`, also its override
             return {}
 
         on_x86 = self.cc_on_x86 or self.cc_on_x64
-        is_unix = self.cc_is_gcc or self.cc_is_clang
+        is_unix = self.cc_is_gcc or self.cc_is_clang or self.cc_is_clang_cl
```

Comment:
`is_unix` will be confusing if it starts including clang-cl on Windows. How about renaming to `is_gcc_or_clang`?

Reply:
I've renamed this.